VRDR/DeathRecord.cs

```diff
@@ -5880,7 +5880,7 @@ public string DateOfDeathPronouncement
         /// <para>// Getter:</para>
         /// <para>Console.WriteLine($"Death Location Jurisdiction: {ExampleDeathRecord.DeathLocationJurisdiction}");</para>
         /// </example>
-        [Property("Death Location Jurisdiction", Property.Types.String, "Death Investigation", "Vital Records Jurisdiction of Death Location.", true, locationJurisdictionExtPath, false, 16)]
+        [Property("Death Location Jurisdiction", Property.Types.String, "Death Investigation", "The Jurisdiction Name, the library will generate the codeable concept.", true, locationJurisdictionExtPath, false, 16)]
```

Comment:
I think we want to still make it clear that this is the location of death; I suggest

```Vital Records Jurisdiction of Death Location (two character jurisdiction code, e.g. CA)```